src/WireMock.Net/Matchers/XPathMatcher.cs

```diff
@@ -19,16 +20,22 @@ namespace WireMock.Matchers;
 /// <seealso cref="IStringMatcher" />
 public class XPathMatcher : IStringMatcher
 {
+    private readonly XmlNamespace[]? _xmlNamespaceMap;
     private readonly AnyOf<string, StringPattern>[] _patterns;
 
     /// <inheritdoc />
     public MatchBehaviour MatchBehaviour { get; }
 
+    /// <summary>
+    /// Array of namespace prefix and uri.
+    /// </summary>
+    public XmlNamespace[]? XmlNamespaceMap => _xmlNamespaceMap;

```

Comment:
when using the same for the public property and the private field, just use 

`public XmlNamespace[]? XmlNamespaceMap { get; private set; }` and remove the private field

Reply:
Done